js/ui/windowManager.js

```diff
@@ -1211,6 +1403,22 @@ const WindowManager = new Lang.Class({
         }
     },
 
+    _rotateInCompleted: function(actor) {
+        if (this._removeEffect(this._rotateInActors, actor)) {
+            Tweener.removeTweens(actor);
+            actor.opacity = 255;
+            actor.rotation_angle_x = 0;
+        }
+    },
+
+    _rotateOutCompleted: function(actor) {
+        if (this._removeEffect(this._rotateOutActors, actor)) {
+            Tweener.removeTweens(actor);
+            actor.hide();
```

Comment:
I would like to make that explicit. In other words, there should always be a way to rotate back to the original application, even without closing builder necessarily.
But yeah, let's save that for another follow up.